connectivity/suite.go

```diff
@@ -824,6 +830,27 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			return check.ResultCurlHTTPError, check.ResultNone // if the header is not set the request will get a 401
 		})
 
+	// Test mTLS auth with always-fail
+	ct.NewTest("echo-ingress-auth-always-fail").WithPolicy(echoIngressAuthFailPolicyYAML).
```

Comment:
#1470 renamed `WithPolicy` to `WithCiliumPolicy`:

```suggestion
	ct.NewTest("echo-ingress-auth-always-fail").WithCiliumPolicy(echoIngressAuthFailPolicyYAML).
```

same fore the other occurrences below.

Reply:
Thanks!